client/src/screens/SignUpScreen.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState, useContext } from "react";
+import {
+    StyleSheet,
+    Text,
+    View,
+    Image,
+    TextInput,
+    TouchableOpacity,
+} from "react-native";
+import AuthService from "../services/auth.service";
+import { SessionContext, IContextType } from "../services/session-context.service";
+
+const data = [
+    { label: 'זכר', value: 'male' },
+    { label: 'נקבה', value: 'female' },
+];
+
+export const SignUpScreen = () => {
+    const context = useContext<IContextType | null>(SessionContext);
+
+    const [email, setEmail] = useState("");
+    const [username, setUsername] = useState("");
+    const [password, setPassword] = useState("");
+    const [age, setAge] = useState(0);
+    const [gender, setGender] = React.useState("");
+    const [isFocus, setIsFocus] = useState(false);
+
+    const onSignUpPress = async () => {
+        try {
+            const userSession = await AuthService.signUp(username, email, password, age, gender);
+            if (context !== null) context.updateSession(userSession);
+        } catch (err) {
+            console.log(JSON.stringify(err));
+        }
+    };
+
+    return (
+    <View style={styles.container}>
+        <Image style={styles.image} source={require("../../assets/logo-spic.png")} />
+        <StatusBar style="auto" />
+        <View style={styles.inputView}>
+            <TextInput
+                style={{...styles.TextInput}}
+                placeholder="אימייל"
+                placeholderTextColor="#003f5c"
+                onChangeText={(email) => setEmail(email)}
+            />
+        </View>
+        <View style={styles.inputView}>
+            <TextInput
+                style={styles.TextInput}
+                placeholder="שם משתמש"
+                placeholderTextColor="#003f5c"
+                onChangeText={(username) => setUsername(username)}
+            />
+        </View>
+        <View style={styles.inputView}>
+            <TextInput
+                style={styles.TextInput}
+                placeholder="סיסמה"
+                placeholderTextColor="#003f5c"
+                secureTextEntry={true}
+                onChangeText={(password) => setPassword(password)}
+            />
+        </View>
+        <View style={styles.inputView}>
+            <TextInput
+                style={styles.TextInput}
+                placeholder="וידוא סיסמה"
+                placeholderTextColor="#003f5c"
+                secureTextEntry={true}
+                onChangeText={(verifyPassword) => verifyPassword === password}
+            />
+        </View>
+        <View style={styles.inputView}>
+            <TextInput
+                style={styles.TextInput}
+                keyboardType='numeric'
+                placeholder="גיל"
+                placeholderTextColor="#003f5c"
+                maxLength={3}
+                onChangeText={(age) => setAge(parseInt(age))}
+            />
+        </View>
+        <Dropdown
```

Comment:
Make all text without `marginLeft` like I told you, same in SignInScreen
Try to have the same color for the placeholder in `TextInput` and `Dropdown`, it doesn't look like that right now